leaderless/node/app/acceptor.py

```diff
@@ -0,0 +1,19 @@
+class Acceptor:
+    def __init__(self):
```

Comment:
These properties are different for every paxos run.
As each bank transaction is a result of different paxos run, we need to store this 3 properties for each operation in the transaction log.